app/assets/stylesheets/partials/_facets.scss

```diff
@@ -1,4 +1,13 @@
+.facet-container {
+  position: sticky;
```

Comment:
It looks like you resolved my initial concern with this approach (at least I can no longer replicate it not scrolling to the bottom of the facet list before getting stuck and being weird).

The way it currently works is nice. Thanks!